Sources/Random/CryptoRandom.swift

```diff
@@ -6,7 +6,11 @@ public final class CryptoRandom: Random {
 
     public func bytes(_ num: Int) -> Bytes {
         var random = Bytes(repeating: 0, count: num)
-        RAND_bytes(&random, Int32(num))
+		guard RAND_bytes(&random, Int32(num)) == 1 else {
+			// If the requested number of random bytes couldn't be read,
+			// we need to fail fast and hard.
+			abort()
```

Comment:
We should throw an error instead of `abort()`. If someone is using this in vapor and doesn't have something like supervisor setup, this will kill their web service.